01-inputs-required.tf

```diff
@@ -27,3 +27,11 @@ variable "project_acr_enabled" {
 variable "global_acr_enabled" {
   default = true
 }
+
+variable "monitor_diagnostic_setting" {
+  default = false
+}
+
+variable "sku_tier" {
+  default = "Free"
```

Comment:
add description with available values? Free or Paid isn't it?

Reply:
Yup, will update

Reply:
```suggestion
  default     = "Free"
  description = "Free or Paid (which includes the uptime SLA)"
```

Reply:
description element it used for generating docs (used in most repoes although apparently not this one)